CHANGELOG.md

```diff
@@ -2,7 +2,11 @@
 Blip is the web front end for YourLoops system.
 It is based on Tidepool Blip 1.27.
 
-## 3.3.0-beta - 2024-02-14
+## 3.4.0-beta - 2024-02-14
+### Engineering use
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (non-blocking):** ‏Also update the packages version to 3.4.0-beta? 👼 

```sh
VERSION=3.4.0-beta npm run bump-version
```